ui/app/templates/components/task-row.hbs

```diff
@@ -38,18 +38,6 @@
     {{/each}}
   </ul>
 </td>
-<td data-test-ports>
-  <ul>
-    {{#let this.task.resources.networks.firstObject as |network|}}
-      {{#each network.ports as |port|}}
-        <li data-test-port>
-          <strong>{{port.name}}:</strong>
-          <a href="http://{{network.ip}}:{{port.port}}" target="_blank" rel="noopener noreferrer">{{network.ip}}:{{port.port}}</a>
-        </li>
-      {{/each}}
-    {{/let}}
-  </ul>
-</td>
```

Comment:
Pretty unimportant but this means the corresponding `tasks.ports` property in `tests/pages/allocations/detail` can be deleted. I feel like there are probably several existing orphaned page objects properties though as we don’t have a good way to detect things we aren’t using anymore.